src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,20 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+public class DeleteCommand extends Command {
+
+    int index;
```

Comment:
Good spot! I should specify the type of access modifier explicitly instead of using the default and making people wonder if it is deliberately or accidentally default.